core-bundle/contao/classes/BackendTemplate.php

```diff
@@ -56,10 +56,7 @@ public function getResponse()
 	{
 		$this->compile();
 
-		$response = parent::getResponse();
-		$response->headers->set('Cache-Control', 'no-cache, no-store');
-
-		return $response->setPrivate();
+		return parent::getResponse();
```

Comment:
If we would merge this, `setPrivate()` would no longer be called. I don't think that is correct, is it?

Reply:
Also since we are sending `no-store`, `private` does not really matter. The `private` response directive indicates that the response can be stored only in a private cache - but `no-store` already defines that the response cannot be stored in _any_ cache.

Reply:
> Responses are already private by default (though I haven't checked why exactly).

Probably due to the [`MakeResponsePrivateListener`](https://github.com/contao/contao/blob/5.3/core-bundle/src/EventListener/MakeResponsePrivateListener.php).